plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/utils.ts

```diff
@@ -23,7 +23,7 @@ export function decompressFromString(input: string, doubleDecode = false): strin
                 finalUncompressed,
             })
         }
-        return finalUncompressed
+        return JSON.parse(finalUncompressed)
```

Comment:
why do we `json.parse` here now?

Reply:
Because arguably we should have been parsing it before. The gzipping will be better with proper json and we always want the parsed version anyways